src/modules/services_manager_service.rb

```diff
@@ -472,6 +483,114 @@ def refresh_services
       services.values.each(&:refresh)
     end
 
+    # Summary of services that have been started
+    #
+    # @return [String]
+    def started_services_summary
+      return "" unless started_services.any?
```

Comment:
NP: again, why not `if` over `unless`?

I mean, 

```ruby
return "" if started_services.empty?
```

Reply:
I prefer to use `#any?` when I asking for "is there any element in the list?". I see it more natural because it is closer to the question "is there any service"? Using the word "empty" requires a mental translation, I mean, you have to understand that you are checking "#empty?" because you want to know if there is no services. With "#any?" that mental translation is not required (at least for me). I am not sure if I have convinced you ;)

Reply:
Actually my mind requires a mental translation when see the keyword `unless`. But your explanation make me thought and you convinced me (more or less :stuck_out_tongue_winking_eye:)

Thank you for your time :)